testdata/table_stats

```diff
@@ -175,16 +179,18 @@ wait-pending-table-stats
 ----
 num-entries: 1
 num-deletions: 1
+num-range-keys: 0
 point-deletions-bytes-estimate: 0
-range-deletions-bytes-estimate: 771
+range-deletions-bytes-estimate: 769
```

Comment:
Curiously, these crept in due to the use of the `testkeys.Comparer`. I need to dig into why exactly, but it seems benign.